### PR TITLE
python3Packages.google-cloud-*: updates

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update python3Packages.google-cloud-* packages.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Looks like that `pytestCheckPhase` is execute twice for some packages and fails for the second run.

```bash
  ====================== 1308 passed, 49 warnings in 28.11s ======================
  Finished executing pytestCheckPhase
  pytestcachePhase
  pytestCheckPhase
  Executing pytestCheckPhase
  rm: cannot remove 'google': No such file or directory
``

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 126015` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>14 packages failed to build:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.google-cloud-automl</li>
    <li>python38Packages.google-cloud-datastore</li>
    <li>python38Packages.google-cloud-error-reporting</li>
    <li>python38Packages.google-cloud-pubsub</li>
    <li>python38Packages.google-cloud-spanner</li>
    <li>python38Packages.google-cloud-translate</li>
    <li>python38Packages.textacy</li>
    <li>python39Packages.google-cloud-automl</li>
    <li>python39Packages.google-cloud-datastore</li>
    <li>python39Packages.google-cloud-error-reporting</li>
    <li>python39Packages.google-cloud-pubsub</li>
    <li>python39Packages.google-cloud-spanner</li>
    <li>python39Packages.google-cloud-translate</li>
  </ul>
</details>
<details>
  <summary>80 packages built:</summary>
  <ul>
    <li>dvc-with-remotes</li>
    <li>json-schema-for-humans (python38Packages.json-schema-for-humans)</li>
    <li>python38Packages.dataclasses-json</li>
    <li>python38Packages.google-cloud-access-context-manager</li>
    <li>python38Packages.google-cloud-appengine-logging</li>
    <li>python38Packages.google-cloud-asset</li>
    <li>python38Packages.google-cloud-bigquery</li>
    <li>python38Packages.google-cloud-bigquery-datatransfer</li>
    <li>python38Packages.google-cloud-bigquery-logging</li>
    <li>python38Packages.google-cloud-bigtable</li>
    <li>python38Packages.google-cloud-container</li>
    <li>python38Packages.google-cloud-dataproc</li>
    <li>python38Packages.google-cloud-dlp</li>
    <li>python38Packages.google-cloud-firestore</li>
    <li>python38Packages.google-cloud-iam</li>
    <li>python38Packages.google-cloud-iam-logging</li>
    <li>python38Packages.google-cloud-iot</li>
    <li>python38Packages.google-cloud-kms</li>
    <li>python38Packages.google-cloud-language</li>
    <li>python38Packages.google-cloud-logging</li>
    <li>python38Packages.google-cloud-monitoring</li>
    <li>python38Packages.google-cloud-org-policy</li>
    <li>python38Packages.google-cloud-os-config</li>
    <li>python38Packages.google-cloud-redis</li>
    <li>python38Packages.google-cloud-runtimeconfig</li>
    <li>python38Packages.google-cloud-secret-manager</li>
    <li>python38Packages.google-cloud-securitycenter</li>
    <li>python38Packages.google-cloud-speech</li>
    <li>python38Packages.google-cloud-storage</li>
    <li>python38Packages.google-cloud-tasks</li>
    <li>python38Packages.google-cloud-texttospeech</li>
    <li>python38Packages.google-cloud-trace</li>
    <li>python38Packages.google-cloud-videointelligence</li>
    <li>python38Packages.google-cloud-vision</li>
    <li>python38Packages.google-cloud-websecurityscanner</li>
    <li>python38Packages.hypothesmith</li>
    <li>python38Packages.libcst</li>
    <li>python38Packages.pathy</li>
    <li>python38Packages.spacy</li>
    <li>python38Packages.spacy-transformers</li>
    <li>python38Packages.typing-inspect</li>
    <li>python39Packages.dataclasses-json</li>
    <li>python39Packages.google-cloud-access-context-manager</li>
    <li>python39Packages.google-cloud-appengine-logging</li>
    <li>python39Packages.google-cloud-asset</li>
    <li>python39Packages.google-cloud-bigquery</li>
    <li>python39Packages.google-cloud-bigquery-datatransfer</li>
    <li>python39Packages.google-cloud-bigquery-logging</li>
    <li>python39Packages.google-cloud-bigtable</li>
    <li>python39Packages.google-cloud-container</li>
    <li>python39Packages.google-cloud-dataproc</li>
    <li>python39Packages.google-cloud-dlp</li>
    <li>python39Packages.google-cloud-firestore</li>
    <li>python39Packages.google-cloud-iam</li>
    <li>python39Packages.google-cloud-iam-logging</li>
    <li>python39Packages.google-cloud-iot</li>
    <li>python39Packages.google-cloud-kms</li>
    <li>python39Packages.google-cloud-language</li>
    <li>python39Packages.google-cloud-logging</li>
    <li>python39Packages.google-cloud-monitoring</li>
    <li>python39Packages.google-cloud-org-policy</li>
    <li>python39Packages.google-cloud-os-config</li>
    <li>python39Packages.google-cloud-redis</li>
    <li>python39Packages.google-cloud-runtimeconfig</li>
    <li>python39Packages.google-cloud-secret-manager</li>
    <li>python39Packages.google-cloud-securitycenter</li>
    <li>python39Packages.google-cloud-speech</li>
    <li>python39Packages.google-cloud-storage</li>
    <li>python39Packages.google-cloud-tasks</li>
    <li>python39Packages.google-cloud-texttospeech</li>
    <li>python39Packages.google-cloud-trace</li>
    <li>python39Packages.google-cloud-videointelligence</li>
    <li>python39Packages.google-cloud-vision</li>
    <li>python39Packages.google-cloud-websecurityscanner</li>
    <li>python39Packages.hypothesmith</li>
    <li>python39Packages.json-schema-for-humans</li>
    <li>python39Packages.libcst</li>
    <li>python39Packages.typing-inspect</li>
    <li>sublime-music</li>
    <li>wal_e</li>
  </ul>
</details>

File: pkgs/development/python-modules/google-cloud-bigtable/default.nix [hunk 1]
```diff
@@ -1,33 +1,55 @@
...
+  ];
+
+  checkInputs = [
+    mock
+    pytestCheckHook
+  ];
 
   checkPhase = ''
+    # Prevent google directory from shadowing google imports
     rm -r google
```
Comment (jonringer) [inline pkgs/development/python-modules/google-cloud-bigtable/default.nix:37]: getting a lot of these failures:

```
  Executing pytestCheckPhase
  rm: cannot remove 'google': No such file or directory
```

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 126015` at 1105ef03 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages marked as broken and skipped:</summary>
  <ul>
    <li>python38Packages.spacy</li>
    <li>python38Packages.spacy-transformers</li>
    <li>python38Packages.textacy</li>
  </ul>
</details>
<details>
  <summary>13 packages failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/cfa7e8f0a167c909d2d9cdc560bf8b51">home-assistant</a></li>
    <li><a href="https://gist.github.com/b624c6d4d848f3fc57eff7dfa27c7fdc">python38Packages.google-cloud-automl</a></li>
    <li><a href="https://gist.github.com/214ca709c1e4bad48d1044277864eacc">python38Packages.google-cloud-datastore</a></li>
    <li><a href="https://gist.github.com/19ca82feecd31a8601b5b1ed8335fb99">python38Packages.google-cloud-error-reporting</a></li>
    <li><a href="https://gist.github.com/e79e96305e17065daec270e96eddbfc1">python38Packages.google-cloud-pubsub</a></li>
    <li><a href="https://gist.github.com/8983f1521f68f2b12376776a511aff2c">python38Packages.google-cloud-spanner</a></li>
    <li><a href="https://gist.github.com/eade142212edde0279c9befdcf74cfd0">python38Packages.google-cloud-translate</a></li>
    <li><a href="https://gist.github.com/84d626d66e2ab7ecaeca2e4b6b24ca32">python39Packages.google-cloud-automl</a></li>
    <li><a href="https://gist.github.com/5e9f4ff124336dcc0113749d1833db2e">python39Packages.google-cloud-datastore</a></li>
    <li><a href="https://gist.github.com/abe3fe21bccd1daae4cb4f22d9f0a3b0">python39Packages.google-cloud-error-reporting</a></li>
    <li><a href="https://gist.github.com/4e67400f3ce2b863a647f298d41e0b5f">python39Packages.google-cloud-pubsub</a></li>
    <li><a href="https://gist.github.com/4a99c317899edf564044804f545015af">python39Packages.google-cloud-spanner</a></li>
    <li><a href="https://gist.github.com/b87cd8d29004aa372bc83bfdf5bbc662">python39Packages.google-cloud-translate</a></li>
  </ul>
</details>
<details>
  <summary>78 packages built successfully:</summary>
  <ul>
    <li>dvc-with-remotes</li>
    <li>json-schema-for-humans (python38Packages.json-schema-for-humans)</li>
    <li>python38Packages.dataclasses-json</li>
    <li>python38Packages.google-cloud-access-context-manager</li>
    <li>python38Packages.google-cloud-appengine-logging</li>
    <li>python38Packages.google-cloud-asset</li>
    <li>python38Packages.google-cloud-bigquery</li>
    <li>python38Packages.google-cloud-bigquery-datatransfer</li>
    <li>python38Packages.google-cloud-bigquery-logging</li>
    <li>python38Packages.google-cloud-bigtable</li>
    <li>python38Packages.google-cloud-container</li>
    <li>python38Packages.google-cloud-dataproc</li>
    <li>python38Packages.google-cloud-dlp</li>
    <li>python38Packages.google-cloud-firestore</li>
    <li>python38Packages.google-cloud-iam</li>
    <li>python38Packages.google-cloud-iam-logging</li>
    <li>python38Packages.google-cloud-iot</li>
    <li>python38Packages.google-cloud-kms</li>
    <li>python38Packages.google-cloud-language</li>
    <li>python38Packages.google-cloud-logging</li>
    <li>python38Packages.google-cloud-monitoring</li>
    <li>python38Packages.google-cloud-org-policy</li>
    <li>python38Packages.google-cloud-os-config</li>
    <li>python38Packages.google-cloud-redis</li>
    <li>python38Packages.google-cloud-runtimeconfig</li>
    <li>python38Packages.google-cloud-secret-manager</li>
    <li>python38Packages.google-cloud-securitycenter</li>
    <li>python38Packages.google-cloud-speech</li>
    <li>python38Packages.google-cloud-storage</li>
    <li>python38Packages.google-cloud-tasks</li>
    <li>python38Packages.google-cloud-texttospeech</li>
    <li>python38Packages.google-cloud-trace</li>
    <li>python38Packages.google-cloud-videointelligence</li>
    <li>python38Packages.google-cloud-vision</li>
    <li>python38Packages.google-cloud-websecurityscanner</li>
    <li>python38Packages.hypothesmith</li>
    <li>python38Packages.libcst</li>
    <li>python38Packages.pathy</li>
    <li>python38Packages.typing-inspect</li>
    <li>python39Packages.dataclasses-json</li>
    <li>python39Packages.google-cloud-access-context-manager</li>
    <li>python39Packages.google-cloud-appengine-logging</li>
    <li>python39Packages.google-cloud-asset</li>
    <li>python39Packages.google-cloud-bigquery</li>
    <li>python39Packages.google-cloud-bigquery-datatransfer</li>
    <li>python39Packages.google-cloud-bigquery-logging</li>
    <li>python39Packages.google-cloud-bigtable</li>
    <li>python39Packages.google-cloud-container</li>
    <li>python39Packages.google-cloud-dataproc</li>
    <li>python39Packages.google-cloud-dlp</li>
    <li>python39Packages.google-cloud-firestore</li>
    <li>python39Packages.google-cloud-iam</li>
    <li>python39Packages.google-cloud-iam-logging</li>
    <li>python39Packages.google-cloud-iot</li>
    <li>python39Packages.google-cloud-kms</li>
    <li>python39Packages.google-cloud-language</li>
    <li>python39Packages.google-cloud-logging</li>
    <li>python39Packages.google-cloud-monitoring</li>
    <li>python39Packages.google-cloud-org-policy</li>
    <li>python39Packages.google-cloud-os-config</li>
    <li>python39Packages.google-cloud-redis</li>
    <li>python39Packages.google-cloud-runtimeconfig</li>
    <li>python39Packages.google-cloud-secret-manager</li>
    <li>python39Packages.google-cloud-securitycenter</li>
    <li>python39Packages.google-cloud-speech</li>
    <li>python39Packages.google-cloud-storage</li>
    <li>python39Packages.google-cloud-tasks</li>
    <li>python39Packages.google-cloud-texttospeech</li>
    <li>python39Packages.google-cloud-trace</li>
    <li>python39Packages.google-cloud-videointelligence</li>
    <li>python39Packages.google-cloud-vision</li>
    <li>python39Packages.google-cloud-websecurityscanner</li>
    <li>python39Packages.hypothesmith</li>
    <li>python39Packages.json-schema-for-humans</li>
    <li>python39Packages.libcst</li>
    <li>python39Packages.typing-inspect</li>
    <li>sublime-music</li>
    <li>wal_e</li>
  </ul>
</details>
<details>
  <summary>10 suggestions:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `grpcio`.
Near pkgs/development/python-modules/google-cloud-translate/default.nix:8:3:
```
  |
8 | , grpcio
  |   ^
```
</li>
    <li>warning: unused-argument

Unused argument: `pythonOlder`.
Near pkgs/development/python-modules/google-cloud-iam/default.nix:5:3:
```
  |
5 | , pythonOlder
  |   ^
```
</li>
    <li>warning: unused-argument

Unused argument: `webapp2`.
Near pkgs/development/python-modules/google-cloud-logging/default.nix:13:3:
```
   |
13 | , webapp2
   |   ^
```
</li>
    <li>warning: unused-argument

Unused argument: `pythonOlder`.
Near pkgs/development/python-modules/google-cloud-appengine-logging/default.nix:10:3:
```
   |
10 | , pythonOlder
   |   ^
```
</li>
    <li>warning: unused-argument

Unused argument: `pythonOlder`.
Near pkgs/development/python-modules/google-cloud-bigquery-logging/default.nix:10:3:
```
   |
10 | , pythonOlder
   |   ^
```
</li>
    <li>warning: unused-argument

Unused argument: `pythonOlder`.
Near pkgs/development/python-modules/google-cloud-iam-logging/default.nix:10:3:
```
   |
10 | , pythonOlder
   |   ^
```
</li>
    <li>warning: unused-argument

Unused argument: `pythonOlder`.
Near pkgs/development/python-modules/google-cloud-access-context-manager/default.nix:1:39:
```
  |
1 | { lib, buildPythonPackage, fetchPypi, pythonOlder, google-api-core }:
  |                                       ^
```
</li>
    <li>warning: unused-argument

Unused argument: `pythonOlder`.
Near pkgs/development/python-modules/google-cloud-org-policy/default.nix:1:39:
```
  |
1 | { lib, buildPythonPackage, fetchPypi, pythonOlder, google-api-core, proto-plus }:
  |                                       ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-patch-comment.md">missing-patch-comment</a>

Consider adding a comment explaining the purpose of this patch on the line preceeding.
Near pkgs/development/python-modules/typing-inspect/default.nix:22:5:
```
   |
22 |     (fetchpatch {
   |     ^
```
</li>
    <li>warning: unused-argument

Unused argument: `pytest`.
Near pkgs/development/python-modules/google-cloud-asset/default.nix:12:3:
```
   |
12 | , pytest
   |   ^
```
</li>
  </ul>
</details>

Note that build failures [may predate](https://github.com/nix-community/hydra-check) this PR, and could be nondeterministic or hardware dependent.
Please exercise your independent judgement. Does something look off? Please file an [issue](https://github.com/rmcgibbo/r-rmcgibbo/issues) or reach out on IRC.

**Review by ruuda — APPROVED** [review]
The `libcst` changes look good to me, I confirmed that it builds. I haven’t looked at the other changes in detail.

File: pkgs/development/python-modules/typing-inspect/default.nix [hunk 2]
```diff
@@ -16,16 +18,26 @@ buildPythonPackage rec {
     sha256 = "1dzs9a1pr23dhbvmnvms2jv7l7jk26023g5ysf0zvnq8b791s6wg";
   };
 
+  patches = lib.optional (pythonAtLeast "3.9") [
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/typing-inspect/default.nix:21]:
```suggestion
  patches = [
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/typing-inspect/default.nix (hunk 2 above)
Comment (fabaff) [reply pkgs/development/python-modules/typing-inspect/default.nix:21]: In the meantime the patch is no longer needed as upstream published a new release.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 126015` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>92 packages built:</summary>
  <ul>
    <li>dvc-with-remotes</li>
    <li>home-assistant</li>
    <li>json-schema-for-humans (python38Packages.json-schema-for-humans)</li>
    <li>python38Packages.dataclasses-json</li>
    <li>python38Packages.google-cloud-access-context-manager</li>
    <li>python38Packages.google-cloud-appengine-logging</li>
    <li>python38Packages.google-cloud-asset</li>
    <li>python38Packages.google-cloud-automl</li>
    <li>python38Packages.google-cloud-bigquery</li>
    <li>python38Packages.google-cloud-bigquery-datatransfer</li>
    <li>python38Packages.google-cloud-bigquery-logging</li>
    <li>python38Packages.google-cloud-bigtable</li>
    <li>python38Packages.google-cloud-container</li>
    <li>python38Packages.google-cloud-dataproc</li>
    <li>python38Packages.google-cloud-datastore</li>
    <li>python38Packages.google-cloud-dlp</li>
    <li>python38Packages.google-cloud-error-reporting</li>
    <li>python38Packages.google-cloud-firestore</li>
    <li>python38Packages.google-cloud-iam</li>
    <li>python38Packages.google-cloud-iam-logging</li>
    <li>python38Packages.google-cloud-iot</li>
    <li>python38Packages.google-cloud-kms</li>
    <li>python38Packages.google-cloud-language</li>
    <li>python38Packages.google-cloud-logging</li>
    <li>python38Packages.google-cloud-monitoring</li>
    <li>python38Packages.google-cloud-org-policy</li>
    <li>python38Packages.google-cloud-os-config</li>
    <li>python38Packages.google-cloud-pubsub</li>
    <li>python38Packages.google-cloud-redis</li>
    <li>python38Packages.google-cloud-runtimeconfig</li>
    <li>python38Packages.google-cloud-secret-manager</li>
    <li>python38Packages.google-cloud-securitycenter</li>
    <li>python38Packages.google-cloud-spanner</li>
    <li>python38Packages.google-cloud-speech</li>
    <li>python38Packages.google-cloud-storage</li>
    <li>python38Packages.google-cloud-tasks</li>
    <li>python38Packages.google-cloud-texttospeech</li>
    <li>python38Packages.google-cloud-trace</li>
    <li>python38Packages.google-cloud-translate</li>
    <li>python38Packages.google-cloud-videointelligence</li>
    <li>python38Packages.google-cloud-vision</li>
    <li>python38Packages.google-cloud-websecurityscanner</li>
    <li>python38Packages.hypothesmith</li>
    <li>python38Packages.libcst</li>
    <li>python38Packages.pathy</li>
    <li>python38Packages.spacy</li>
    <li>python38Packages.spacy-transformers</li>
    <li>python38Packages.textacy</li>
    <li>python39Packages.dataclasses-json</li>
    <li>python39Packages.google-cloud-access-context-manager</li>
    <li>python39Packages.google-cloud-appengine-logging</li>
    <li>python39Packages.google-cloud-asset</li>
    <li>python39Packages.google-cloud-automl</li>
    <li>python39Packages.google-cloud-bigquery</li>
    <li>python39Packages.google-cloud-bigquery-datatransfer</li>
    <li>python39Packages.google-cloud-bigquery-logging</li>
    <li>python39Packages.google-cloud-bigtable</li>
    <li>python39Packages.google-cloud-container</li>
    <li>python39Packages.google-cloud-dataproc</li>
    <li>python39Packages.google-cloud-datastore</li>
    <li>python39Packages.google-cloud-dlp</li>
    <li>python39Packages.google-cloud-error-reporting</li>
    <li>python39Packages.google-cloud-firestore</li>
    <li>python39Packages.google-cloud-iam</li>
    <li>python39Packages.google-cloud-iam-logging</li>
    <li>python39Packages.google-cloud-iot</li>
    <li>python39Packages.google-cloud-kms</li>
    <li>python39Packages.google-cloud-language</li>
    <li>python39Packages.google-cloud-logging</li>
    <li>python39Packages.google-cloud-monitoring</li>
    <li>python39Packages.google-cloud-org-policy</li>
    <li>python39Packages.google-cloud-os-config</li>
    <li>python39Packages.google-cloud-pubsub</li>
    <li>python39Packages.google-cloud-redis</li>
    <li>python39Packages.google-cloud-runtimeconfig</li>
    <li>python39Packages.google-cloud-secret-manager</li>
    <li>python39Packages.google-cloud-securitycenter</li>
    <li>python39Packages.google-cloud-spanner</li>
    <li>python39Packages.google-cloud-speech</li>
    <li>python39Packages.google-cloud-storage</li>
    <li>python39Packages.google-cloud-tasks</li>
    <li>python39Packages.google-cloud-texttospeech</li>
    <li>python39Packages.google-cloud-trace</li>
    <li>python39Packages.google-cloud-translate</li>
    <li>python39Packages.google-cloud-videointelligence</li>
    <li>python39Packages.google-cloud-vision</li>
    <li>python39Packages.google-cloud-websecurityscanner</li>
    <li>python39Packages.hypothesmith</li>
    <li>python39Packages.json-schema-for-humans</li>
    <li>python39Packages.libcst</li>
    <li>sublime-music</li>
    <li>wal_e</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 126015` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>2 packages failed to build and are new build failures:</summary>
  <ul>
<li>python39Packages.google-cloud-access-context-manager: log was empty</li>
<li>python39Packages.google-cloud-asset: log was empty</li>
  </ul>
</details>
<details>
<summary>
90 packages built:
</summary>
<ul>
<li>dvc-with-remotes</li>
<li>home-assistant</li>
<li>json-schema-for-humans (python38Packages.json-schema-for-humans)</li>
<li>python38Packages.dataclasses-json</li>
<li>python38Packages.google-cloud-access-context-manager</li>
<li>python38Packages.google-cloud-appengine-logging</li>
<li>python38Packages.google-cloud-asset</li>
<li>python38Packages.google-cloud-automl</li>
<li>python38Packages.google-cloud-bigquery</li>
<li>python38Packages.google-cloud-bigquery-datatransfer</li>
<li>python38Packages.google-cloud-bigquery-logging</li>
<li>python38Packages.google-cloud-bigtable</li>
<li>python38Packages.google-cloud-container</li>
<li>python38Packages.google-cloud-dataproc</li>
<li>python38Packages.google-cloud-datastore</li>
<li>python38Packages.google-cloud-dlp</li>
<li>python38Packages.google-cloud-error-reporting</li>
<li>python38Packages.google-cloud-firestore</li>
<li>python38Packages.google-cloud-iam</li>
<li>python38Packages.google-cloud-iam-logging</li>
<li>python38Packages.google-cloud-iot</li>
<li>python38Packages.google-cloud-kms</li>
<li>python38Packages.google-cloud-language</li>
<li>python38Packages.google-cloud-logging</li>
<li>python38Packages.google-cloud-monitoring</li>
<li>python38Packages.google-cloud-org-policy</li>
<li>python38Packages.google-cloud-os-config</li>
<li>python38Packages.google-cloud-pubsub</li>
<li>python38Packages.google-cloud-redis</li>
<li>python38Packages.google-cloud-runtimeconfig</li>
<li>python38Packages.google-cloud-secret-manager</li>
<li>python38Packages.google-cloud-securitycenter</li>
<li>python38Packages.google-cloud-spanner</li>
<li>python38Packages.google-cloud-speech</li>
<li>python38Packages.google-cloud-storage</li>
<li>python38Packages.google-cloud-tasks</li>
<li>python38Packages.google-cloud-texttospeech</li>
<li>python38Packages.google-cloud-trace</li>
<li>python38Packages.google-cloud-translate</li>
<li>python38Packages.google-cloud-videointelligence</li>
<li>python38Packages.google-cloud-vision</li>
<li>python38Packages.google-cloud-websecurityscanner</li>
<li>python38Packages.hypothesmith</li>
<li>python38Packages.libcst</li>
<li>python38Packages.pathy</li>
<li>python38Packages.spacy</li>
<li>python38Packages.spacy-transformers</li>
<li>python38Packages.textacy</li>
<li>python39Packages.dataclasses-json</li>
<li>python39Packages.google-cloud-appengine-logging</li>
<li>python39Packages.google-cloud-automl</li>
<li>python39Packages.google-cloud-bigquery</li>
<li>python39Packages.google-cloud-bigquery-datatransfer</li>
<li>python39Packages.google-cloud-bigquery-logging</li>
<li>python39Packages.google-cloud-bigtable</li>
<li>python39Packages.google-cloud-container</li>
<li>python39Packages.google-cloud-dataproc</li>
<li>python39Packages.google-cloud-datastore</li>
<li>python39Packages.google-cloud-dlp</li>
<li>python39Packages.google-cloud-error-reporting</li>
<li>python39Packages.google-cloud-firestore</li>
<li>python39Packages.google-cloud-iam</li>
<li>python39Packages.google-cloud-iam-logging</li>
<li>python39Packages.google-cloud-iot</li>
<li>python39Packages.google-cloud-kms</li>
<li>python39Packages.google-cloud-language</li>
<li>python39Packages.google-cloud-logging</li>
<li>python39Packages.google-cloud-monitoring</li>
<li>python39Packages.google-cloud-org-policy</li>
<li>python39Packages.google-cloud-os-config</li>
<li>python39Packages.google-cloud-pubsub</li>
<li>python39Packages.google-cloud-redis</li>
<li>python39Packages.google-cloud-runtimeconfig</li>
<li>python39Packages.google-cloud-secret-manager</li>
<li>python39Packages.google-cloud-securitycenter</li>
<li>python39Packages.google-cloud-spanner</li>
<li>python39Packages.google-cloud-speech</li>
<li>python39Packages.google-cloud-storage</li>
<li>python39Packages.google-cloud-tasks</li>
<li>python39Packages.google-cloud-texttospeech</li>
<li>python39Packages.google-cloud-trace</li>
<li>python39Packages.google-cloud-translate</li>
<li>python39Packages.google-cloud-videointelligence</li>
<li>python39Packages.google-cloud-vision</li>
<li>python39Packages.google-cloud-websecurityscanner</li>
<li>python39Packages.hypothesmith</li>
<li>python39Packages.json-schema-for-humans</li>
<li>python39Packages.libcst</li>
<li>sublime-music</li>
<li>wal_e</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 126015` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>python39Packages.google-cloud-access-context-manager</li>
<li>python39Packages.google-cloud-asset</li>
</ul>
</details>

**Pull request merged**